Tests/XCUITests/HistoryTests.swift

```diff
@@ -282,7 +282,7 @@ class HistoryTests: BaseTestCase {
         } else {
             waitForExistence(app.navigationBars.staticTexts["Open Tabs"])
         }
-        XCTAssertFalse(app.staticTexts[bookOfMozilla["title"]!].exists)
+        XCTAssertFalse(app.staticTexts[bookOfMozilla["title"]!].isHittable)
```

Comment:
There is a `CollectionView` for each private tabs and regular tabs. In addition, there's no label on the `CollectionView` to tell the difference between private and public. However, the one that's not visible ("Book of Mozilla" in this case) is not hittable.

I don't like this solution, but it works.

Reply:
@isabelrios If you know a better solution to test if something is not visible currently, please let me know. 🙂